xapian-core/include/xapian/weight.h

```diff
@@ -415,7 +429,7 @@ class XAPIAN_VISIBILITY_DEFAULT TfIdfWeight : public Weight {
 
     /* When additional normalizations are implemented in the future, the additional statistics for them
        should be accessed by these functions. */
-    double get_wdfn(Xapian::termcount wdf, char c) const;
+    double get_wdfn(Xapian::termcount wdf, char c, Xapian::termcount wdfdocmax = 0) const;
```

Comment:
A minor point, but the default parameter value here doesn't seem to be used anywhere, so I'd just drop it.